### ISSUE TITLE
ADD NEXT EPISODE BUTTON FOR DOWNLOADED FILE

### ISSUE THREAD

**Issue by Krishna-art-dev** [recloudstream/cloudstream#1023]
### Describe your suggested feature

There Is No Way To Play next ep in cs3 when playing download file, we need to go back to download list and play from there also the player itself didn't play next ep when current ep is being completed

add a button for in the video player for download file that let us play next ep or previous ep from that player

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Luna712** [comment]
Just to note, I have a local patch working for this, But I will push to a PR once my currently open PR is merged in order to prevent conflicts.
